x-pack/plugins/fleet/common/services/simplified_package_policy_helper.ts

```diff
@@ -5,7 +5,6 @@
  * 2.0.
  */
 
-import { packageToPackagePolicy } from '.';
```

Comment:
hmmm looks like there were some merge problems in here,  since I haven't touched any of these fleet files